README.md

```diff
@@ -1,32 +1,153 @@
 
-## RUN
+## How to run
+
+You'll need to first install the dependencies with:
 
 ```bash
+npm install
+```
 
-# Start
-npm run build
-npm run start
+Then you can do
+
+```bash
+npm run start # it'll build the project first
 ```
 
 ## API
 
-Based on https://github.com/decentraland/adr/blob/main/docs/ADR-42-third-party-assets-integration.md#third-party-resolver.
+Once again we follow the [docs](https://docs.decentraland.org/decentraland/linked-wearables/#building-the-api) here for each endpoint we expose. We'll explain each one briefly, check the docs for more info and [ADR](https://github.com/decentraland/adr/blob/main/docs/ADR-42-third-party-assets-integration.md#third-party-resolver) for more info
 
 ### @GET /registry/:registry-id/address/:address/assets
 
+This endpoint is responsible of returning all assets an address own, for a particular registry. It has a [response format](https://github.com/decentraland/adr/blob/main/docs/ADR-42-third-party-assets-integration.md#get-registryregistry-idaddressaddressassets) it has to follow.
+
 #### Example
 
-```
-https://third-party-resolver-api.decentraland.io/v1/registry/pepe/address/0x123/assets
+```bash
+# Try to find assets for an invalid registry
+/v1/registry/invalidregistry/address/0xc04528c14c8ffd84c7c1fb6719b4a89853035cdd/assets
+{
+  "address": "0xc04528c14c8ffd84c7c1fb6719b4a89853035cdd",
+  "total": 0,
+  "page": 1,
+  "next": "",
+  "assets": []
+}
+
+# Try to find assets for an address that has none
+/v1/registry/cryptoregistry/address/0x1bb1c46af05fed1a407d86c468a783d13a1acf7e/assets
+{
+  "address": "0x1bb1c46af05fed1a407d86c468a783d13a1acf7e",
+  "total": 0,
+  "page": 1,
+  "next": "",
+  "assets": []
+}
+
+# Try to find assets for a valid registry and a owner that exists
+/v1/registry/cryptoregistry/address/0xc04528c14c8ffd84c7c1fb6719b4a89853035cdd/assets
+
+{
+  "address": "0xc04528c14c8ffd84c7c1fb6719b4a89853035cdd",
+  "total": 1,
+  "page": 1,
+  "next": "",
+  "assets": [
+    {
+      "id": "0xb794f5ea0ba39494ce839613fffba74279579268:2",
+      "amount": 1,
+      "urn": {
+        "decentraland": "urn:decentraland:matic:collections-thirdparty:cryptoregistry:0xb794f5ea0ba39494ce839613fffba74279579268:2"
+      }
+    }
+  ]
+}
 ```
 
 ### @GET /registry/:registry-id/address/:address/assets/:id
 
+This endpoint is responsible of returning a single asset via it's id. It must belong to the regsitry and address. It has a [response format](https://github.com/decentraland/adr/blob/main/docs/ADR-42-third-party-assets-integration.md#get-registryregistry-idaddressaddressassetsid) it has to follow.
+
 #### Example
 
+```bash
+# Try to find an asset for an invalid registry
+/v1/registry/invalidthings/address/0xc04528c14c8ffd84c7c1fb6719b4a89853035cdd/assets/0xb794f5ea0ba39494ce839613fffba74279579268:2
+{
+  "id": "0xb794f5ea0ba39494ce839613fffba74279579268:2",
+  "amount": 0,
+  "urn": {
```

Comment:
I thought that the response was `"urn": null` but I am ok if this is better

Reply:
If I'm being honest I couldn't find an example in neither the [ADR](https://github.com/decentraland/adr/blob/main/docs/ADR-42-third-party-assets-integration.md#third-party-resolver) nor the [docs](https://docs.decentraland.org/decentraland/linked-wearables/#building-the-api) so I went with the types that were on the project already 😊

Reply:
Do you mind adding it to the ADR & Docs once this is merged?